src/lex.c

```diff
@@ -24,7 +24,7 @@ static const StringMappingType TOKEN_MAP[] = {
    { "Button",             TOK_BUTTON           },
    { "ButtonClose",        TOK_BUTTONCLOSE      },
    { "ButtonMax",          TOK_BUTTONMAX        },
-   { "ButtonMaxActive",    TOK_BUTTONMAXACTIVE  },
+   { "ButtonMaximized",    TOK_BUTTONMAXIMIZED  },
```

Comment:
I think this breaks backwards compatibility? For something like this, I think it should support both options unless there is a major release or something.

As far as the name is concerned, the button isn't maximized, it's pressed. So I might think it should be renamed to "ButtonMaxPressed". Though, given the backwards compatibility problem, I'm not convinced it's a worthwhile change.

Reply:
Makes sense, but I'm still concerned about the backward compatibility issue. Some way of naming this such that it didn't break the backwards compatibility I think would be ideal. If that's not possible, I guess I should probably push through 2.4.4 and save this for 2.5.0.

Reply:
Sorry for not getting back to you sooner. I think using "Focus" would be fine, obviously different than other places, but I think "Focus" is more precise and probably what should be used in the other places anyway since "Active" is kind of ambiguous (hence its appearance here).